### PR TITLE
fix #6095 GC2CJPF archived causing tests to fail

### LINKED ISSUE #6095: GC2CJPF has been archived: causing some tests to fail
Cache https://coord.info/GC2CJPF has been archived and causing these tests to fail:

```
21:29:55  
21:29:55  cgeo.geocaching.CgeoApplicationTest > testSearchByGeocodeBasis[emulator(AVD) - 6.0] [31mFAILED [0m
21:29:55  	org.junit.ComparisonFailure: [Cache GC2CJPF: archived] expected:<[fals]e> but was:<[tru]e>
21:29:55  	at cgeo.test.Compare.assertCompareCaches(Compare.java:25)
21:29:59  
21:29:59  cgeo.geocaching.CgeoApplicationTest > testSearchByGeocodeNotLoggedIn[emulator(AVD) - 6.0] [31mFAILED [0m
21:29:59  	java.lang.AssertionError:
21:29:59  	Actual and expected should have same size but actual size is:
21:30:06  
21:30:06  cgeo.geocaching.CgeoApplicationTest > testSearchByViewport[emulator(AVD) - 6.0] [31mFAILED [0m
21:30:06  	org.junit.ComparisonFailure: expected:<[tru]e> but was:<[fals]e>
21:30:06  	at cgeo.geocaching.CgeoApplicationTest$6.run(CgeoApplicationTest.java:317)
21:30:15  
21:30:15  cgeo.geocaching.connector.gc.GCBaseTest > testSearchFromMap[emulator(AVD) - 6.0] [31mFAILED [0m
21:30:15  	org.junit.ComparisonFailure: [Cache GC2CJPF: disabled] expected:<[fals]e> but was:<[tru]e>
21:30:15  	at cgeo.test.Compare.assertCompareCaches(Compare.java:24)
21:31:45  :main:connectedBasicDebugAndroidTest FAILED

```

Now we have only archived Mocked Caches for tests. I think it would be good to have a new active Cache added to our Mocked Caches for testing.

### PR DESCRIPTION
**Pull request by pstorch**
Using GC3FJ5F as a replacement for GC2JPF for some tests, because the latter has been archived.

### CODE REVIEW

File: tests/src-android/cgeo/geocaching/test/mock/GC3FJ5F.java [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    }
+
+    @Override
+    public String getLocation() {
+        return "Nordrhein-Westfalen, Germany";
+    }
+
+    @Override
+    public boolean isFound() {
+        if ("blafoo".equals(this.getMockedDataUser())) {
```
Comment (Lineflyer) [inline tests/src-android/cgeo/geocaching/test/mock/GC3FJ5F.java]: Is "blafoo" here still correct?

File: tests/src-android/cgeo/geocaching/test/mock/GC3FJ5F.java [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        if ("Lineflyer".equals(Settings.getUserName())) {
+            return true;
+        }
+        return super.isOwner();
+    }
+
+    @Override
+    public boolean isFavorite() {
+        if ("blafoo".equals(this.getMockedDataUser())) {
+            return true;
```
Comment (Lineflyer) [inline tests/src-android/cgeo/geocaching/test/mock/GC3FJ5F.java]: See above

File: tests/src-android/cgeo/geocaching/test/mock/GC3FJ5F.java (hunk 1 above)
Comment (pstorch) [inline tests/src-android/cgeo/geocaching/test/mock/GC3FJ5F.java]: Actually, just copied from the other Mocked Cache. I'll try without.

Comment (pstorch) [inline tests/src-android/cgeo/geocaching/test/mock/GC3FJ5F.java]: Works without, I'll remove it.

**Pull request merged**